AsBuiltReport.Microsoft.AD.json

```diff
@@ -1,9 +1,9 @@
 {
     "Report": {
         "Name": "Microsoft AD As Built Report",
-        "Version": "1.0",
+        "Version": "0.2.0",
         "Status": "Released",
-        "ShowCoverPageImage": true,
+        "ShowCoverPageImage": false,
```

Comment:
This should default to `true`, as users may use their own custom style scripts. The default style script should not include the image content if it is not supported.

Reply:
Done!